model/src/main/kotlin/Snippet.kt

```diff
@@ -49,7 +49,7 @@ data class Snippet(
     /**
      * The license of the component the code snippet is commit from.
      */
-    val licenses: SpdxExpression,
+    val license: SpdxExpression,
```

Comment:
`license` sound like only a single license, IMO `licenseExpression` or `spdxExpression` would be clearer. But we also have `license: SpdxExpression` in `LicenseFinding`, so maybe just adjust the docs to state that this is an SPDX expression?

Reply:
> But we also have `license: SpdxExpression` in `LicenseFinding`

Yes, that's where I took inspiration.

> so maybe just adjust the docs to state that this is an SPDX expression?

Done. While at it, I've also improved / aligned the docs in `LicenseFinding` a bit.